staging/src/k8s.io/apiserver/pkg/endpoints/watch_test.go

```diff
@@ -878,7 +879,7 @@ func BenchmarkWatchHTTP(b *testing.B) {
 		item.Name = fmt.Sprintf("reasonable-name-%d", i)
```

Comment:
nit: self-links were removed, may or may not be useful to remove these, perhaps in some follow-up